test/functional/test_runner.py

```diff
@@ -528,6 +528,10 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
 
     # Test Framework Tests
     print("Running Unit Tests for Test Framework Modules")
+
+    tests_dir = src_dir + '/test/functional/'
+    sys.path.append(tests_dir)
```

Comment:
Suggested documentation since it's not immediately obvious why this is necessary.
```suggestion
    # This allows `test_runner.py` to work from an out-of-source build directory using a symlink,
    # hard link or a copy on any platform. See https://github.com/bitcoin/bitcoin/pull/27561.
    sys.path.append(tests_dir)
```

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/27561#issuecomment-1555065798).